### DISCUSSION TITLE
FreeType library

### DISCUSSION THREAD

**Discussion by ironsightdesign** [micropython/micropython#15249] [Libraries & Drivers] [answered]
@peterhinch Trying to get fonts converted to python using micropython-font-to-py. I've installed freetype with brew install freetype and it's listed in brew list, but when I try to run the command to convert, I get font_to_py requires freetype library. Any ideas?

 https://github.com/peterhinch/micropython-font-to-py

**Comment by peterhinch** [comment]
Please see [the docs: Dependencies](https://github.com/peterhinch/micropython-font-to-py/blob/master/FONT_TO_PY.md#2-dependencies).

**Reply by ironsightdesign** [reply]
I got it to work with Windows. Couldnt do anything to get the script to recognize freetype on a mac.

**Reply by peterhinch** [reply]
On the mac what happens at Python 3 REPL if you issue
```py
>>> import freetype
```
?
On Linux:
```py
$ python3
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import freetype
>>> 
```
I have no mac experience but I have been assured that `font-to-py.py` runs.

**Comment by ponch0v** [comment]
Im sure it does and Im just doing something wrong. python3 and freetype are both installed. 
<img width="527" alt="Screenshot 2024-06-11 at 3 20 41 PM" src="https://github.com/micropython/micropython/assets/10678695/7e7dfd96-6690-4a85-b29f-a737546fcb41">

**Reply by bikeNomad** [reply]
homebrew's `freetype` is just the library itself. You need the Python module as well. Try this (assuming you don't have a Python `venv` set up):
```
pip3 install freetype-py --break-system-packages --user
```

**Reply by ponch0v** [reply, marked as answer]
Yep...I reinstalled pip and then freetype_py and font_to_py is working. 

Thanks!

**Comment by peterhinch** [comment]
If the Mac requires specific installation instructions I'd be very grateful for a docs PR explaining them :)
